client/modules/reconportal/src/LeafletMap/LeafletMap.tsx

```diff
@@ -228,6 +265,18 @@ export const LeafletMap: React.FC = () => {
         </MarkerClusterGroup>
         {/* Zoom control */}
         <ZoomControl position="topright" />
+
+        {/* Selected event banner popup */}
+        {selectedEvent && showSelectedPopup && (
+          <Popup
+            position={[selectedEvent.location.lat, selectedEvent.location.lon]}
+            offset={[0, -10]}
+            closeButton={false}
+            closeOnClick={false}
+          >
+            <ReconPortalPopup dataset={selectedEvent} />
```

Comment:
For the selected event, I think they want a title above the pin instead of the full popup.
<img width="319" height="185" alt="Screenshot 2025-07-18 at 10 58 19 AM" src="https://github.com/user-attachments/assets/9f1742a4-3304-450e-bf99-2b62fb48a388" />

https://docs.google.com/document/d/1FHb1sn_axaYZTaliDaM7ZcBr7yv-QzjrDESyfT5xJaE/edit?tab=t.0